docs/source/create/avatars/package-avatar.rst

```diff
@@ -6,9 +6,9 @@
 Package and Host Your Avatar
 ##################################
 
-At a minimum, avatars in High Fidelity must have an FBX model and an associated FST file that includes information about how your avatar looks and behaves. Together, these two files (with any optional texture or script) form an "avatar package". There are two ways you can create an avatar package: by using the `Avatar Packager`_ in Interface or the `High Fidelity Avatar Exporter for Unity`_ in Unity.
+At a minimum, avatars in Vircadia must have an FBX, glTF, or GLB model, and an associated FST file that includes information about how your avatar looks and behaves. Together, these two files (with any optional texture or script) form an "avatar package". There are two ways you can create an avatar package: by using the `Avatar Packager`_ in Interface or the `High Fidelity Avatar Exporter for Unity`_ in Unity.
```

Comment:
```suggestion
At a minimum, avatars in Vircadia must have an FBX, glTF, or GLB model, and an associated FST file that includes information about how your avatar looks and behaves. Together, these two files (with any optional texture or script) form an "avatar package". There are two ways you can create an avatar package: by using the `Avatar Packager`_ in Interface or the `Vircadia Avatar Exporter for Unity`_ in Unity.
```

Reply:
Just a warning, this package is not being actively maintained by us (but anyone is free to take it on if they want to.)

Reply:
You link to the "Vircadia Avatar Exporter" but further up you still talk about "The High Fidelity Avatar Exporter for Unity"

Reply:
Fixed.